integration/spark2/src/main/java/org/apache/carbondata/spark/vectorreader/VectorizedCarbonRecordReader.java

```diff
@@ -121,12 +126,24 @@ public VectorizedCarbonRecordReader(QueryModel queryModel, InputMetricsStats inp
       while (ext != null) {
         if (ext instanceof FileNotFoundException) {
           throw new InterruptedException(
-              e.getMessage() + ". insert overwrite may be in progress.Please check");
+              "Insert overwrite may be in progress.Please check " + e.getMessage());
```

Comment:
use new InterruptedException (msg, e) to add `e` itself to the InterruptedException object

Reply:
InterruptedException takes only one parameter